spec/support/star_wars/data.rb

```diff
@@ -39,7 +39,7 @@ module StarWars
 
   ActiveRecord::Schema.define do
     self.verbose = false
-    create_table :bases do |t|
+    create_table :bases, force: true do |t|
```

Comment:
I added this to make it easier to re-run tests.  Ideally though we'd create the table and destroy it after the tests had run.  I can remove this if its not a good idea.